content/english/blog/blog-post-20220401.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Conversational AI Mailing List"
+date: 2022-04-01T16:50:01+01:00
+author: Ricardo G. Sousa
+image : "images/blog/blog-post-8-mailing-list.jpg"
+bg_image: "images/featue-bg.jpg"
+categories: ["Dissemination"]
+tags: ["mailing list", "share", "communication"]
+draft: false
+type: "post"
+---
+
+
+
+We've launched a [Conversational AI mailing list](https://www.freelists.org/list/conversational-ai) as part of our commitment to allow for more accessible communication on a crucial topic: Conversational AI Agents.
+
+The lack of mailings that share, discuss, and brainstorm the development, business, and research problems when designing and implementing conversational agents has been noted as a massive gap.
```

Comment:
Instead of "noted" could be "identified"? It feels more natural

Reply:
fixed. thanks for the feedback. 3be5d08